domains/thrasher.json

```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Discord Turkey",
+    "owner": {
+        "username": "thrashxr",
+        "email": "ajtroag@gmail.com"
+    },
+    "record": {
+        "CNAME": "thrashxr.github.io"
```

Comment:
404 GitHub Page. Doesn't match the vercel domain you provided with your preview.

See docs: https://docs.is-a.dev/vercel